### PR TITLE
Add links to speakers' profile

### PR DESCRIPTION
**Pull request by Ploypiti**
Issue **#126** [Add link to speakers' profile](https://github.com/fossasia/2016.fossasia.org/issues/126)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -106,27 +109,27 @@
...
 							</div>
 
 							<div class="widget">
 								<ul class="social-profiles">
-									<li><a href="https://twitter.com/fossasia" target="default"><i class="icon social_twitter"></i></a></li>
-									<li><a href="https://facebook.com/fossasia" target="default"><i class="icon social_facebook"></i></a></li>
-									<li><a href="http://fossasia.org" target="default"><i class="icon social_dribbble"></i></a></li>
-									<li><a href="https://www.flickr.com/photos/fossasia/" target="default"><i class="icon social_instagram"></i></a></li>
-									<li><a href="https://plus.google.com/108920596016838318216" target="default"><i class="icon social_googleplus"></i></a></li>
+									<li><a target="_self" href="http://twitter.com/fossasia"><i class="icon social_twitter"></i></a></li>
```
Comment (mariobehling) [inline index.html:123]: why are you changing this? Should not be changed.

**Review by mariobehling — CHANGES_REQUESTED** [review]
Too many changes, many are good, but a number of changes are not suitable. You are changing the drop down menu links for other events. They are correct. As you are changing a lot of things that are not related to the speakers profile we cannot accept this PR. You aren't doing what the issue and the description says. Please submit a clean PR that is related to each change and make it easier to review by sticking exactly to what is mentioned in an issue.

**Comment by Ploypiti** [comment]
I _did not_ change anything besides adding link to speakers' profile and tidy up the code (it messed up when I add the code) I wonder _when_ did I change the dropdown menu?

**Comment by mariobehling** [comment]
Please tidy up the code in a separate PR.

**Comment by Ploypiti** [comment]
I know, I'll send a not-tidied code later as I can't use my laptop right
now, however I insist I didn't make any change beyond the task.

Regards

Ploypiti

**Review by akshatnitd — CHANGES_REQUESTED** [review]
I can see that you have made a whole lot of changes, out of which many are not required. I would suggest you to close this PR, sync your code with original repo, make changes and submit a fresh PR :)

**Pull request closed without merging**